doc/quick-start.md

```diff
@@ -15,22 +15,24 @@ kernelspec:
 
 # Quick Start
 
-JupySQL allows you to run SQL in Jupyter/IPython via a `%sql` and `%%sql` magics. It is a fork of `ipython-sql` with bug fixes and a lot of great new features!
+JupySQL allows you to run SQL and plot large datasets in Jupyter via a `%sql`, `%%sql`, and `%sqlplot` magics. JupySQL is compatible with all major databases (e.g., PostgreSQL, MySQL, SQL Server), data warehouses (e.g., Snowflake, BigQuery, Redshift), and embedded engines (SQLite, and DuckDB).
+
+It is a fork of `ipython-sql` with many bug fixes and a lot of great new features!
 
 +++
 
 ## Installation
 
-Run this on your terminal:
+Run this on your terminal (we'll use DuckDB for this example):
 
 ```sh
-pip install jupysql
+pip install jupysql duckdb-engine
```

Comment:
Don't we want to tie at least the quickstart to a local sqllite?

Reply:
It was tied to sqlite in the previous version. But sqlite is incompatible with the new plotting features, so I switched it to duckdb.